### PR TITLE
Gramma fix #14297

### LINKED ISSUE #14297: Message from blocking stunbaton with shield is wrong
"The Muhammed Al-Is-Nis's stunbaton sparks harmlessly off Unknown's shields!"

Message from blocking stunbaton with shield doesn't have a span class and has an incorrect "The"

### PR DESCRIPTION
**Pull request by Gartley**
Fixes #14297

Added span class and reworded message while trying to keep the flavour
of text.

NB: I know this is a very small fix, but I wanted to make sure I was
using GIT correctly before digging any deeper.

### CODE REVIEW

**Comment by Gartley** [comment]
Sorry, massive error on my part. I thought it would have just included my branch, not the whole fork.

File: code/hub.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #else
 	hub_password = "SORRYNOPASSWORD"
 #endif
-	name = "/tg/ Station 13"
+	name = "Cocktail Party Station 13"
```
Comment (KazeEspada) [inline code/hub.dm]: Tell me more about this cocktail party?

Comment (Gartley) [inline code/hub.dm]: A small group of folks that play and mod games. The reason for the fork was to create a SS13 catering to our play style. However was planning on sharing any fixes to bugs in the shared code base. Hence the grammar fix, I'm new to GitHub and SVNs, hence the balls up. Sorry.

File: code/_globalvars/configuration.dm [hunk 2]
```diff
@@ -4,7 +4,7 @@ var/datum/protected_configuration/protected_config = null
 var/host = null
 var/join_motd = null
 var/station_name = null
-var/game_version = "/tg/ Station 13"
+var/game_version = "CP Station 13"
```
Comment (duncathan) [inline code/_globalvars/configuration.dm]: MODS MODS MODS

**Comment by Gartley** [comment]
Those are not mods, ;) these are mods. http://steamcommunity.com/id/gartley/myworkshopfiles/

File: code/_globalvars/configuration.dm (hunk 2 above)
Comment (Gartley) [inline code/_globalvars/configuration.dm]: Those are not mods, ;) these are mods. http://steamcommunity.com/id/gartley/myworkshopfiles/

**Pull request closed without merging**